client/idl/manifest.json

```diff
@@ -2101,8 +2101,12 @@
           },
           {
             "name": "Reverse"
+          },
+          {
+            "name": "ReverseTight"
           }
         ]
+        ]
```

Comment:
?

Reply:
typo. we lost our client generating once we merged certora so been doing this manual